src/TimingData.cpp

```diff
@@ -15,6 +15,17 @@ static const int INVALID_INDEX = -1;
 
 TimingSegment* GetSegmentAtRow( int iNoteRow, TimingSegmentType tst );
 
+// According to The C++ Programming Language 3rd Ed., decreasing the size
```

Comment:
I'll need to research this one a bit. You're referring to a book made in 1997 when we're using c++11 now.